Dynamic/FormFieldTypeInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Sulu\Bundle\FormBundle\Dynamic;
+
+use Sulu\Bundle\FormBundle\Entity\FormField;
+use Symfony\Component\Form\FormBuilderInterface;
+
+interface FormFieldTypeInterface
+{
+    /**
+     * @return string
+     */
+    public function getAlias();
```

Comment:
do we really need that? i would try to avoid that. the alias should be only configured if you want to reuse a type class.